08_cell_line_prediction/plot_stratified_drug_response.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Probably can remove this cell :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Haha, will do!</p>